nixos/modules/config/users-groups.nix

```diff
@@ -704,6 +704,11 @@ in {
     in stringAfter [ "users" ] ''
       if [ -e ${lingerDir} ] ; then
         cd ${lingerDir}
+        for user in ${lingerDir}/*; do
+          if ! id "$user" >/dev/null 2>&1; then
```

Comment:
You should not pipe stderr to /dev/null here, as it hides failures. This PR broke lingering for me. 

```
Mar 21 13:24:18 bank2 stage-2-init: id: '/var/lib/systemd/linger/*': no such user
Mar 21 13:24:18 bank2 stage-2-init: System has not been booted with systemd as init system (PID 1). Can't operate.
Mar 21 13:24:18 bank2 stage-2-init: Failed to connect to bus: Host is down
Mar 21 13:24:18 bank2 stage-2-init: Activation script snippet 'update-lingering' failed (123
```